packages/playground-examples/copy/pt/JavaScript/External APIs/TypeScript com React.tsx

```diff
@@ -0,0 +1,128 @@
+//// { order: 2, compiler: { jsx: 2, esModuleInterop: true } }
+
+//React é uma biblioteca muito popular para criar interfaces de usuário.
+//Ela providencia uma abstração do Javascript ao criar visualização de componentes
+//utilizando uma extensão da linguagem, chamada JSX.
+
+//TypeScript suporta JSX, e providencia um amplo conjunto de ferramentas de 
+//tipagem que modela ricamente como os componentes se conectam.
+
+//Para entender como o TypeScript funciona com os componentes do React
+//podemos começar com exemplos genéricos:
+//
+
+//-exemplo: funções-genéricas
+//-exemplo: classes-genéricas
```

Comment:
This is a syntax for a hidden hyperlink.
See the example in EN. I think you may want to keep it the same.
```suggestion
// - example:generic-functions
// - example:generic-classes
```